run-prod

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash -xe
-
-
-BINARY=$1
-
-CFGDIR=$PWD
-cd $PWD/mozilla-central/testing/tps || exit 1
-
-/bin/rm -rf $HOME/tps-prod
-./create_venv.py $HOME/tps-prod
-. $HOME/tps-prod/bin/activate
-# xvfb-run - uses DISPLAY=:99 (by default)
-xvfb-run runtps --debug --binary="${BINARY}" --testfile=test_sync.js --configfile=$CFGDIR/prod-config.json
```

Comment:
@rpappalax: This was the line we were discussing earlier as well (re: Issue #3). We'll need to pass in some env var (a la `$TEST_CONFIG`) which we can use to specify the specific `--testfile`. Not sure if a pseudo-Boolean `IS_PROD=true` is good enough, or if that'd be super weird if you passed in a stage config with `IS_PROD=true` on the CLI. 🤔 

I may try reading up on whether we can have 2 entry points in our docker file so we can just specify whether we want to run against stage or prod.